release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -115,6 +115,10 @@ xref:../installing/installing_restricted_networks/installing-restricted-networks
 and xref:../operators/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
 for details.
 
+[id="ocp-4-2-three-node-bare-metal-deployments"]
+==== Three-node bare metal deployments (Technology Preview)
+In {product-title} {product-version}, the ability to deploy a cluster without distinct compute, or worker nodes, on bare metal clusters is a technology preview. If you do not create worker machines when you install your cluster on bare metal infrastructure that you provision, all pods, including the router pods, are deployed on the control plane, or master, machines instead. Note that this deployment method is not available in cloud-based clusters in {product-title} {product-version}. It is not recommended to use technology preview features in production clusters.
```

Comment:
s/is a technology preview/a Technology Preview feature 
or
s/is a technology preview/in Technology Preview

We treat TP as a proper noun. You may also want to add an xref to ocp-4-2-technology-preview, which further defines TP and lists all TP features.

s/pods/Pods